### ISSUE TITLE
add "games" section

### ISSUE THREAD

**Issue by regulator-g** [mpsq/arewewaylandyet#110]
Hi there
this site is great for tracking Wayland progress

can you please add a section relating to gaming, perhaps covering Steam and Lutris and how games work with Wayland as the display server. 

cheers!

**Comment by mpsq** [comment]
This is a good idea, unfortunately I do not run any games on Linux myself and my knowledge is limited in the area. Please submit a PR if you can :).

**Comment by ChrisJAllan** [comment]
Also a sub-section for VR, which last time I heard only worked with some WIP patches

**Comment by regulator-g** [comment]
Yeah, i was kind of thinking we can't just have a page saying "steam supports Wayland" we need a page which instructs people what config changes need to be made so that their games run natively on Wayland.. The idea applies to other apps too which have a Wayland setting which must be activated 

@mpsq should we make a sub page with steps to activate Wayland for certain games?
